core/corehttp/ipns_hostname.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 
 	"github.com/ipfs/go-ipfs/core"
+	opts "github.com/ipfs/go-ipfs/namesys/opts"
```

Comment:
Either drop prefix or add to the import above too

Reply:
Done.